### PR TITLE
[JENKINS-22316] Show candidates containing the input value

### PR DESCRIPTION
**Pull request by ikedam**
Update for #35, fixing the code comment.

### CODE REVIEW

File: src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js [hunk 1]
```diff
@@ -51,11 +51,13 @@ Behaviour.register({"SELECT.staticCombobox": function(e) {
     
     /*
      * Changes from the original 3:
-     *   Original behavior: Show candidates that matches the current incomplete input.
-     *   Changed behavior : Show all the candidates, not concerning with the current incomplete input.
+     *   Original behavior: Show candidates that start with the current incomplete input.
```
Comment (cupsadarius) [inline src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js]: @ikedam The original behaviour would return all items regardless of the input box, other than that it's fine.

Comment (ikedam) [reply src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js]: In this context, the original one means `<f:combobox>`.

Comment (cupsadarius) [reply src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js]: oh, ok then it makes sense.

**Comment by ikedam** [comment]
I plan to make a new release after working for https://issues.jenkins-ci.org/browse/JENKINS-49210

**Comment by cupsadarius** [comment]
@ikedam sounds good!

**Pull request merged**